### PR TITLE
Feature / notification lambda + timestream

### PR DESCRIPTION
**Pull request by vdektiarev**
1. Added trigger for notification lambda
2. Added Timestream DB
3. Extracted Timestream to a separate CF stack
4. Renamed infra folders and stacks.

### CODE REVIEW

**Review by NastassiaPaskrobka — APPROVED** [review]

File: infra_monitoring_account/tests/unit/test_infra_monitoring_stack.py [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import aws_cdk as core
 import aws_cdk.assertions as assertions
 
-from infra_monitoring.infra_monitoring_stack import InfraMonitoringStack
+from infra_monitoring_account.infra_monitoring_stack import InfraMonitoringStack
```
Comment (natmasslove) [inline infra_monitoring_account/tests/unit/test_infra_monitoring_stack.py:4]: Maybe, it'd be better to change name to "infra_monitorED_account" (both, folder and stack names) - as we discussed earlier.

File: infra_tooling_account/infra_tooling_account/infra_tooling_alerting_stack.py [hunk 2]
```diff
@@ -45,25 +51,40 @@ def __init__(self, scope: Construct, construct_id: str, **kwargs) -> None:
             event_bus_name=f"eventbus-{project_name}-notification-{stage_name}"
         )
 
+        # Notification Lambda Eventbridge rule
+        notification_lambda_event_rule = events.Rule(
```
Comment (natmasslove) [inline infra_tooling_account/infra_tooling_account/infra_tooling_alerting_stack.py:55]: 1. I believe this rule should be in monitored account (where events are generated)
2. let's make pattern more specific
  aws:glue, ... <- services we CAN handle (+ step functions, + (?) lambda if it's possible, + (?) glue workflow if it's separate from glue itsleft)

File: infra_monitoring_account/tests/unit/test_infra_monitoring_stack.py (hunk 1 above)
Comment (vdektiarev) [reply infra_monitoring_account/tests/unit/test_infra_monitoring_stack.py]: Fixed

File: infra_tooling_account/infra_tooling_account/infra_tooling_alerting_stack.py (hunk 2 above)
Comment (vdektiarev) [reply infra_tooling_account/infra_tooling_account/infra_tooling_alerting_stack.py:55]: Discussed - leaving as it should be

**Pull request merged**